presto-cassandra/pom.xml

```diff
@@ -17,8 +17,9 @@
 
     <dependencies>
         <dependency>
-            <groupId>com.facebook.presto.cassandra</groupId>
-            <artifactId>cassandra-driver</artifactId>
+            <groupId>com.datastax.cassandra</groupId>
+            <artifactId>cassandra-driver-core</artifactId>
```

Comment:
Is this a drop in replacement for the presto jar? Are both shaded jars?

Reply:
This is not shaded jar , this is actual datastax jar . cassandra-driver-core internally was using datastax 3.6.0 and no further update was done . so we removed that dependancy and used the datastax jar directly

Reply:
Presto and watsonx.data are using the below presto-cassandra driver for Cassandra https://mvnrepository.com/artifact/com.facebook.presto.cassandra/cassandra-driver/3.6.0-1

This is a shaded jar that has DataStax driver bundled inside

com.datastax.cassandra cassandra-driver-core 3.6.0 runtime
It is using 3.6.0 version of DataStax driver which does not support SSL. Version 3.11.2 of DataStax driver supports SSL.

Reply:
Have you tested this version with the community version of Presto itself?

Reply:
yeah I have tested the version with the community code and it worked fine